prereise/gather/griddata/hifld/data_process/transmission.py

```diff
@@ -668,12 +686,33 @@ def build_transmission(method="sub2line", **kwargs):
         lines, substations, **island_kwargs
     )
 
+    # Separate DC lines from AC line processing, add ratings
+    ac_lines, dc_lines = split_lines_to_ac_and_dc(lines, const.dc_line_ratings.keys())
+    dc_lines["Pmax"] = dc_lines.index.to_series().map(const.dc_line_ratings)
+    dc_lines["Pmin"] = -1 * dc_lines["Pmax"]
+
     # Add voltages to lines with missing data
-    augment_line_voltages(lines, substations)
+    augment_line_voltages(ac_lines, substations)
+
+    # Create buses from lines
+    bus = create_buses(ac_lines)
```

Comment:
Not sure whether we should return `bus` data frame as well, otherwise we will need to call `create_buses` in the higher-level orchestration function again using `branch` as inputs, right?

Reply:
Good catch, done.